cmd/bots/internal/telegram/handlers/handlers.go

```diff
@@ -97,6 +97,9 @@ func removeCmd(
 
 func addAliasCmd(env *common.TelegramBotEnvironment, requestType chan<- pair.Request) func(c telebot.Context) error {
 	return func(c telebot.Context) error {
+		if !env.IsEligibleForAction(strconv.FormatInt(c.Chat().ID, 10)) {
+			return c.Send("Sorry, you have no right to change subscriptions")
+		}
```

Comment:
I think it's better to create a telebot middleware and use it in handlers initialization block.